.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -300,7 +300,7 @@ upload_minimized_btfs_arm64:
   script:
     - echo "s3://dd-pulumi-state?region=us-east-1&awssdk=v2&profile=$AWS_PROFILE" > $STACK_DIR
     - pulumi login $(cat $STACK_DIR | tr -d '\n')
-    - inv -e kmt.gen-config --ci --arch=$ARCH --output-file=$VMCONFIG_FILE --sets=$TEST_SETS
+    - inv -e kmt.gen-config --ci --arch=$ARCH --output-file=$VMCONFIG_FILE --sets=$TEST_SETS --host-cpus=$AVAILABLE_CPUS
```

Comment:
This would return the number of processors on the gitlab runner. We want to provide the available CPUs on the remote ec2 instances. It is impossible to do it dynamically at this stage because the machine has not been launched.

Reply:
Oops, you're right, forget it then :D